lib/currency/index.js

```diff
@@ -32,41 +46,44 @@ var symbols = {
     format: '$0,0.00',
     raw: '0,0.00'
   }
-};
+}; // eslint-disable-next-line no-unused-vars
 
 function parseLocaleNumber(stringNumber) {
   var thousandSeparator = 1111 .toLocaleString().replace(/1/g, '');
   var decimalSeparator = 1.1.toLocaleString().replace(/1/g, '');
-  return parseFloat(stringNumber.replace(new RegExp('\\' + thousandSeparator, 'g'), '').replace(new RegExp('\\' + decimalSeparator), '.'));
-}
+  return parseFloat(stringNumber.replace(new RegExp("\\".concat(thousandSeparator), 'g'), '').replace(new RegExp("\\".concat(decimalSeparator)), '.'));
+} // eslint-disable-next-line import/prefer-default-export
+
 
 function setLocale() {
   var locale = arguments.length > 0 && arguments[0] !== undefined ? arguments[0] : 'USD';
```

Comment:
Wow is this hokey or what?

This isn't you @corydeppen but I hate this whole f1-utils library. We should've just used a library that did this and some template strings.